core/arch/arm/mm/core_mmu.c

```diff
@@ -52,6 +52,16 @@
 
 #include "core_mmu_private.h"
 
+#ifndef DEBUG_XLAT_TABLE
+#define DEBUG_XLAT_TABLE 0
+#endif
+
+#if DEBUG_XLAT_TABLE
```

Comment:
I don't think this is needed, the `dump_xlat_table()` that prints is already surrounded by `#if DEBUG_XLAT_TABLE`.
Could use `DMSG_RAW()` directly in that function instead.